dspace-server-webapp/src/test/java/org/dspace/app/rest/AuthorizationRestRepositoryIT.java

```diff
@@ -804,6 +805,11 @@ public void findByObjectInternalServerErrorTest() throws Exception {
      * 
      * @throws Exception
      */
+    // This test currently doesn't work as expected since the AuthorizationFeatureRestRepository#findOne method
+    // is only exposed to admins. Currently we're performing checks on the individual REST objects with its findOne
+    // Permission constraints, which is ADMIN in this case. Seeing as we're trying to retrieve it with a normal
+    // EPerson token in the second test, this will fail.
+    @Ignore
```

Comment:
this should be possible now
see https://github.com/DSpace/DSpace/pull/2778/files#diff-311a7e07c257d9c869f4de1347f5f761R51

Reply:
Removing the `@Ignore` does indeed work now